src/encoded/static/components/graph.js

```diff
@@ -650,8 +650,16 @@ export class Graph extends React.Component {
             newValue = newValue < 0 ? 0 : 100;
         }
         this.slider.current.value = newValue.toString();
-        const event = new Event('input', { bubbles: true });
-        this.slider.current.dispatchEvent(event);
+
+        if (typeof (Event) === 'function') {
+            const event = new Event('input', { bubbles: true });
+            this.slider.current.dispatchEvent(event);
+        } else {
+            // Needed for IE11
+            const event = document.createEvent('Event', { bubbles: true });
+            event.initEvent('input', true, false, window, 0);
+            this.slider.current.dispatchEvent(event);
+        }
```

Comment:
Just curious, could we just use the code that works for IE11 for all the browsers? Or is it worth keeping both methods because eventually we'll want to drop support for IE11 and the other method is better?

Reply:
If you insist I can, as they both would work in all browsers. The thinking of the approach I used is that Event-approach is the newer way while document.createEvent is an older way. See https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Creating_and_triggering_events. So, some would say it is okay to have both so as not to force new browsers to use the old way. But again, if you insist, I can use the old way for all browsers.

Reply:
I think it's better the way you have it! Hopefully we can eventually drop the older method.